content/service.go

```diff
@@ -30,20 +28,28 @@ type RequestParams struct {
 	toDateEpoch   int64
 }
 
-func NewContentByConceptService(conn neoutils.NeoConnection) ContentByConceptServicer {
-	return ConceptService{conn}
+type neoReadStruct struct {
+	UUID  string   `json:"uuid"`
+	Types []string `json:"types"`
 }
 
-func (cd ConceptService) Check() error {
-	return neoutils.Check(cd.conn)
+func NewContentByConceptService(neoURL string, neoConf neoutils.ConnectionConfig) (*ConceptService, error) {
+	conn, err := neoutils.Connect(neoURL, &neoConf)
+	if err != nil {
+		return nil, fmt.Errorf("could not connect to Neo4j: %w", err)
+	}
+	return &ConceptService{conn}, nil
 }
 
-type neoReadStruct struct {
-	UUID  string   `json:"uuid"`
-	Types []string `json:"types"`
+func (cd *ConceptService) CheckConnection() (string, error) {
+	err := neoutils.Check(cd.conn)
+	if err != nil {
+		return "Could not connect to database!", err
+	}
+	return "", nil
 }
 
-func (cd ConceptService) GetContentForConcept(conceptUUID string, params RequestParams) (contentList, bool, error) {
+func (cd *ConceptService) GetContentForConcept(conceptUUID string, params RequestParams) ([]Content, error) {
 	var results []neoReadStruct
```

Comment:
neoReadStruct is used only here now. Seems clearer to me if we use anonymous struct here:
```suggestion
        var results []struct {
                UUID  string   `json:"uuid"`
                Types []string `json:"types"`
        }
```